Cargo.toml

```diff
@@ -15,7 +15,7 @@ edition = "2018"
 libc = "0.2.50"
 
 [features]
-check_pipe = []
+do_not_check_pipe = []
```

Comment:
This is also not a good idea since features should be additive.

With this do_not_check_pipe feature,if one of the crate enables this, then  every other crate is opt out of checking.

Reply:
Otherwise if one enables check then one uses smth different from pipe will fail.

Reply:
If checking or not checking the pipe-ness should be configurable, then it should probably be a runtime option passed to `from_env`.

Not checking is the GNU Make behavior, checking catches errors in practice.